### ISSUE TITLE
 Prototype Pollution in async < 3.2.2

### ISSUE THREAD

**Issue by rlsf** [rollbar/rollbar.js#1021]
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution in async                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ async                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.2.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ rollbar                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ rollbar > async                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://github.com/advisories/GHSA-fwr7-v2mv-hh25            │
└───────────────┴──────────────────────────────────────────────────────────────┘
```
npm audit reports rollbar as vulnerable due to its dependency on async v1.2.1.

**Comment by robinsturmeit** [comment]
This has been fixed in `main` already (it's on `3.2.3` in `package.json`), but the question remains when there will be a release with the change

**Comment by waltjones** [comment]
Released in 2.25.0. https://github.com/rollbar/rollbar.js/releases/tag/v2.25.0
